vitrina/models.py

```diff
@@ -5,1421 +5,1423 @@
+#     created = models.DateTimeField(blank=True, null=True)
+#     modified = models.DateTimeField(blank=True, null=True)
+#     version = models.IntegerField()
+#     # dataset_id = models.BigIntegerField(blank=True, null=True)
+#     # user_id = models.BigIntegerField(blank=True, null=True)
+#     deleted = models.TextField(blank=True, null=True)  # This field type is a guess.
+#     deleted_on = models.DateTimeField(blank=True, null=True)
+#     dataset = models.ForeignKey(Dataset, models.DO_NOTHING, db_column='dataset', blank=True, null=True, related_name = "datasets")
+#     user = models.ForeignKey(User, models.DO_NOTHING, db_column='user', blank=True, null=True, related_name = "userId")
+#     active = models.TextField()  # This field type is a guess.
+#
+#     class Meta:
+#         managed = True
+#         db_table = 'user_dataset_subscription'
+#
+#
+# class UserLike(models.Model):
+#     id = models.BigAutoField(primary_key=True)
+#     created = models.DateTimeField(blank=True, null=True)
+#     deleted = models.TextField(blank=True, null=True)  # This field type is a guess.
+#     deleted_on = models.DateTimeField(blank=True, null=True)
+#     modified = models.DateTimeField(blank=True, null=True)
+#     version = models.IntegerField()
+#     request_id = models.BigIntegerField(blank=True, null=True)
+#     user_id = models.BigIntegerField(blank=True, null=True)
+#
+#     class Meta:
+#         managed = True
+#         db_table = 'user_like'
+#
+#
+# class UserTablePreferences(models.Model):
+#     id = models.BigAutoField(primary_key=True)
+#     created = models.DateTimeField(blank=True, null=True)
+#     deleted = models.TextField(blank=True, null=True)  # This field type is a guess.
+#     deleted_on = models.DateTimeField(blank=True, null=True)
+#     modified = models.DateTimeField(blank=True, null=True)
+#     version = models.IntegerField()
+#     table_column_string = models.TextField(blank=True, null=True)
+#     table_id = models.CharField(max_length=255, blank=True, null=True)
+#     user_id = models.BigIntegerField(blank=True, null=True)
+#
+#     class Meta:
+#         managed = True
+#         db_table = 'user_table_preferences'
+#
+#
+# class UserVote(models.Model):
+#     id = models.BigAutoField(primary_key=True)
+#     created = models.DateTimeField(blank=True, null=True)
+#     deleted = models.TextField(blank=True, null=True)  # This field type is a guess.
+#     deleted_on = models.DateTimeField(blank=True, null=True)
+#     modified = models.DateTimeField(blank=True, null=True)
+#     version = models.IntegerField()
+#     rating = models.IntegerField()
+#     dataset = models.ForeignKey(Dataset, models.DO_NOTHING, blank=True, null=True)
+#     user = models.ForeignKey(User, models.DO_NOTHING, blank=True, null=True)
+#     harvested = models.ForeignKey(HarvestingResult, models.DO_NOTHING, blank=True, null=True)
+#
+#     class Meta:
+#         managed = True
+#         db_table = 'user_vote'
```

Comment:
Could you please just remove all the commented out code? We don't need comment out code, since all the code and all the changes are available under version control and we can restore everything we need.

Thanks.